CONTRIBUTING.md

```diff
@@ -1,24 +1,134 @@
+<!-- markdownlint-disable MD043 MD041 -->
+# Table of contents <!-- omit in toc -->
+
+- [Contributing Guidelines](#contributing-guidelines)
+    - [Reporting Bugs/Feature Requests](#reporting-bugsfeature-requests)
+    - [Contributing via Pull Requests](#contributing-via-pull-requests)
+        - [Dev setup](#dev-setup)
+        - [Local documentation](#local-documentation)
+    - [Conventions](#conventions)
+        - [General terminology and practices](#general-terminology-and-practices)
+        - [Testing definition](#testing-definition)
+    - [Finding contributions to work on](#finding-contributions-to-work-on)
+    - [Code of Conduct](#code-of-conduct)
+    - [Security issue notifications](#security-issue-notifications)
+    - [Troubleshooting](#troubleshooting)
+        - [API reference documentation](#api-reference-documentation)
```

Comment:
Looks that we don't have `API reference doc` section so it can be removed from this content list.